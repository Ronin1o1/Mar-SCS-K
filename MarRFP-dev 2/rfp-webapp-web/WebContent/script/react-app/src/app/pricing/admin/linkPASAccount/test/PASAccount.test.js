import React from "react";
import { mount,render,fireEvent,getAllByRole,find,shallow } from "enzyme";
import toJson from "enzyme-to-json";
import PASAccount from "../content/PASAccount";
import CSelect from "../../../../common/components/CSelect";

it("renders without crashing", () => {
  mount(<PASAccount />);
});

it("renders correctly", () => {
  const tree = mount(<PASAccount />);
  expect(toJson(tree)).toMatchSnapshot();
});

it("should component check header", () => {
  const wrapper = shallow(<PASAccount />);  
  expect(wrapper.text()).toEqual("<PASAccountContextProvider />");
});

it("should component check header", () => {
  const wrapper = shallow(<PASAccount />);  
  expect(wrapper.text()).toHaveLength(29);
});

it("renders PAS Account Component", () => {
  const wrapper = mount(<PASAccount />);
  expect(wrapper.find(<CSelect />))
 
});







// it("should component check all input field component", () => {
//   const wrapper = mount(<PASAccount />);
//   const ShowAll = wrapper.find("#field_Name");
//   expect(ShowAll.text()).toEqual("PAS Manager");
// });


