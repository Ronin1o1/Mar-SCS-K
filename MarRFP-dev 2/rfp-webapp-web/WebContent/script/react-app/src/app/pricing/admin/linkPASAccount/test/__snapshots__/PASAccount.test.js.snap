// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<PASAccount>
  <PASAccountContextProvider>
    loading
  </PASAccountContextProvider>
</PASAccount>
`;
